anxcloud/resource_virtual_server_test.go

```diff
@@ -568,3 +580,51 @@ func createNewNetworkInterface(info environment.Info) vm.Network {
 		IPs:     []string{info.Prefix.GetNextIP()},
 	}
 }
+
+func mockedTemplateList() []templates.Template {
+	return []templates.Template{
+		{ID: "e9325be9-25b9-468e-851e-56b5c0367e5a", Name: "Ubuntu 21.04", Build: "b72"},
+		{ID: "b21b8b77-30e3-478a-9b6d-1f61d29e9f9a", Name: "Flatcar Linux Stable", Build: "b73"},
+		{ID: "ec547552-d453-42e6-987d-51abe703c439", Name: "Debian 11", Build: "b18"},
+		{ID: "26a47eee-dc9a-4eea-b67a-8fb1baa2fcc0", Name: "Flatcar Linux Stable", Build: "b74"},
+		{ID: "cb16dc94-ec55-4e9a-a1a3-b76a91bbe274", Name: "Windows 2022", Build: "b06"},
+		{ID: "fc3a63c6-6f4e-4193-b368-ebe9e08b4302", Name: "Debian 10", Build: "b80"},
+		{ID: "844ac596-5f62-4ed2-936e-b99ffe0d4f88", Name: "Flatcar Linux Stable", Build: "b72"},
+		{ID: "c3d4f0a6-978a-49fb-a952-7361bf531e4f", Name: "Debian 9", Build: "b92"},
+		{ID: "086c5f99-1be6-46ec-8374-cdc23cedd6a4", Name: "Windows 2022", Build: "b12"},
```

Comment:
those test cases will sadly break over time, as templates get lost

maybe checking for "something was found", retrieving the found template by its ID and comparing some values is enough?

Reply:
ah they are fake, alright then :)